QueAns/models.py

```diff
@@ -0,0 +1,64 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+class Meetings(models.Model):
+    HOMER = 1
+    PLATO = 2
+    JARVIS = 3
+    FRIDAY = 4
+    VENUE_CHOICES = (
+        (HOMER,'homer'),
+        (PLATO,'plato'),
+        (JARVIS,'jarvis'),
+        (FRIDAY,'friday'),
+    )
+    venue = models.SmallIntegerField(choices=VENUE_CHOICES,default=HOMER)
+    moderator = models.ForeignKey(User, on_delete=models.CASCADE)
+    start_time = models.DateTimeField(help_text="Start time of the meeting")
+    end_time = models.DateTimeField(help_text="End time of the meeting")
+
+class Question(models.Model):
+    YES = 1
+    NO = 2
+    ANON_CHOICES = (
+        (YES, 'yes'),
+        (NO, 'no'),
+    )
+    question_text = models.CharField(max_length=300)
+    meeting_id = models.ForeignKey(Meetings,on_delete=models.CASCADE)
+    asker = models.ForeignKey(User,on_delete=models.CASCADE)
+    anon_status = models.SmallIntegerField(choices=ANON_CHOICES,default=NO)
+
+    def __str__(self):
+        if self.anon_status == 1:
+            return "%s MEETING ID: %s AUTHOR:ANONYMOUS" % (self.question_text,self.meeting_id)
+        else:
+            return '%s MEETING ID: %s  AUTHOR: %s' % (self.question_text,self.meeting_id,self.asker)
+
+class Message(models.Model):
+    user_id = models.ForeignKey(User,on_delete=models.CASCADE)
+    text = models.CharField(max_length = 1000)
+
+class Voting(models.Model):
+    UPVOTE = 1
+    DOWNVOTE = 2
+    NONE = 3
```

Comment:
What does None voting mean?

Reply:
Just a default choice if User doesn't vote on the question.